docs/shared/contribution_guidelines/vocabulary.rst

```diff
@@ -4,10 +4,15 @@ Vocabulary
 
 If you want to contribute to openMINDS by adding information to the openMINDS vocabulary we advise you to first review this chapter.
 
+For each contribution, please raise an `ISSUE <https://github.com/openMetadataInitiative/openMINDS/issues>`_. Start the issue title with ``vocab: `` followed by an informative short title for your contribution. Please provide a more detailed explanation of your contribution in issue description. We will discuss your contribution with you before a pull request is made (by you or by us if you do not feel comfortable making a pull request yourself).
+
+
 Schema types
 ############
 
-The following code block sketches the vocabulary implementation for a single schema type:
+In openMINDS, schema types are considered version independent, meaning the general information about a schema type remains the same across version (e.g. it's description). We maintain this schema type information centrally on the main openMINDS GitHub repository under: `vocab/types.json <https://github.com/openMetadataInitiative/openMINDS/blob/main/vocab/types.json>`_
+
+This file is a simple JSON, where each vocabulary implementation for a single schema type is structured in the following way:
```

Comment:
We should avoid using "JSON" as a noun in formal text. Suggestion:

"In this JSON file, each vocabulary implementation for a single schema type is structured in the following way:"

Reply:
done